plugins/orchestrator/meta_service.go

```diff
@@ -35,6 +35,13 @@ import (
 	"go.ligato.io/vpp-agent/v3/proto/ligato/generic"
 )
 
+var reservedLabelKeys = map[string]struct{}{
```

Comment:
I think we can omit enforcing of the reserved label keys. Let's just define some guidelines for the key format. I would simply use same guidelines as the [Docker Labels](https://docs.docker.com/config/labels-custom-metadata/#key-format-recommendations):

> ### Key format recommendations
> 
> A label key is the left-hand side of the key-value pair. Keys are alphanumeric strings which may contain periods (`.`) and hyphens (`-`). Most Docker users use images created by other organizations, and the following guidelines help to prevent inadvertent duplication of labels across objects, especially if you plan to use labels as a mechanism for automation.
> 
> - Authors of third-party tools should prefix each label key with the reverse DNS notation of a domain they own, such as `com.example.some-label`.
> 
> - Do not use a domain in your label key without the domain owner’s permission.
> 
> - The `com.docker.*`, `io.docker.*`, and `org.dockerproject.*` namespaces are reserved by Docker for internal use.
> 
> - Label keys should begin and end with a lower-case letter and should only contain lower-case alphanumeric characters, the period character (`.`), and the hyphen character (`-`). Consecutive periods or hyphens are not allowed.
> 
> - The period character (`.`) separates namespace “fields”. Label keys without namespaces are reserved for CLI use, allowing users of the CLI to interactively label Docker objects using shorter typing-friendly strings.
> 
> These guidelines are not currently enforced and additional guidelines may apply to specific use cases.

Where the keys for internal use would be `io.ligato.*` and `ligato.*`